docs/releases/1.10.3.txt

```diff
@@ -23,3 +23,6 @@ Bugfixes
 
 * Fixed ``QuerySet.bulk_create()`` on PostgreSQL when the number of objects is
   a multiple plus one of ``batch_size`` (:ticket:`27385`).
+
+* Fixed redirecting issue when session destroyed during request processing
```

Comment:
Maybe something like "Replaced a possibly incorrect redirect when a session is destroyed in a concurrent request with a `SuspiciousOperation`."

Actually, I'm not sure if SuspiciousOperation is really the best since that generally goes to logging. Maybe it's fine, just wanted to raise the point for consideration.